core/templates/core/post-report.html

```diff
@@ -0,0 +1,58 @@
+{% extends 'base.html' %}
+{% load static %}
+{% load crispy_forms_tags %}
+
+{% block extra_css %}
+<link rel="stylesheet" href="{% static 'core/css/style.css' %}">
+{% endblock %}
+
+{% block content %}
+    <div class="container mt-5">
+        <h1 class="mb-4">Dodaj zgłoszenie postu</h1>
+        <div class="card bg-dark shadow-sm mb-3 text-light">
+
+            <div class="col">
+                <div class="d-flex flex-column">
+                    <div>
+                        <a href="#"
+                           class="link-primary link-offset-2 link-underline-opacity-0 link-underline-opacity-25-hover">
+                            r/{{ post.community }}
+                        </a>
+                        • <small class="text-secondary">{{ post.created_at|timesince }} ago</small>
+                    </div>
+                    <div>
+                        <a href="#" class="link-light link-offset-2 link-underline-opacity-0 link-opacity-75-hover link-underline-opacity-25-hover">
+                            {{ post.author.nickname }}
+                        </a>
+                    </div>
+                </div>
+            </div>
+            <div class="card-body">
+                <h3 class="h3">{{ post.title }}</h3>
+                {{ post.content }}
+            </div>
+        </div>
+        {% if form.errors %}
```

Comment:
Do You see an issue here?
Please look in detail: What condition you are checking here, and what is being looped over in line 38?
Hint: We had that discussion in previous forms too.